src/snowcli/cli/snowpark/function/commands.py

```diff
@@ -221,7 +222,8 @@ def function_update(
             replace = True
         elif (
             resource_json["handler"].lower() != handler.lower()
-            or resource_json["returns"].lower() != return_type.lower()
+            or sql_to_python_return_type_mapper(resource_json["returns"])
```

Comment:
IMO we shouldn't assume that the util function will always return lowercased type name.

```suggestion
            or sql_to_python_return_type_mapper(resource_json["returns"]).lower()
```

Reply:
Oh, but it will. The return of mapping function is:
`return mapping.get(resource_return_type.lower(), resource_return_type.lower())`

Reply:
I know it does it at the moment.
But it's only because you know about it now.
What if I want to add a new mapping and I don't know about the convention that all values of dict should be in lowercase?
What if I decide to completely change implementation of `sql_to_python_return_type_mapper` and I don't know that it has extra responsibility of converting the result to lowercase?

Reply:
Ok, good point. Done